sos/report/__init__.py

```diff
@@ -680,6 +681,21 @@ def _get_network_namespaces(self):
                 out_ns.append(line.partition(' ')[0])
         return out_ns
 
+    def _get_devices_by_fstype(self):
+        _dev_fstypes = {}
+        _devs = sos_get_command_output("lsblk -nrpo FSTYPE,NAME")
+        if _devs['status'] != 0:
+            return _dev_fstypes
+        for line in (_devs['output'].splitlines()):
+            helper = line.strip().split()
+            if len(helper) == 1:
+                helper.insert(0, 'unknown')
+            if "ext" in helper[0]:
+                helper[0] = 'ext2/3/4'
```

Comment:
Can we not be more specific here? If not, I think we should just put `ext` rather than slashed version possibilities.

Reply:
ACK will change it to ext there is no conflict with other blkid types 

```
blkid -k | grep ext
xfs_external_log
ext4dev
ext4
ext3
ext2
```

Reply:
Left it as ext4